.NET/Microsoft.Recognizers.Text.DataDrivenTests/NumberWithUnit/TestNumberWithUnit_German.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using Microsoft.Recognizers.Text.DataDrivenTests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Recognizers.Text.NumberWithUnit.Tests
+{
+    [TestClass]
+    public class TestNumberWithUnit_German : TestBase
+    {
+        public static TestResources TestResources { get; protected set; }
+        public static IDictionary<string, IModel> Models { get; protected set; }
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext context)
+        {
+            TestResources = new TestResources();
+            TestResources.InitFromTestContext(context);
+            Models = new Dictionary<string, IModel>();
+        }
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            base.TestSpecInitialize(TestResources);
+            base.ModelInitialize(Models);
+        }
+
+        //[DataSource("Microsoft.VisualStudio.TestTools.DataSource.CSV", "AgeModel-German.csv", "AgeModel-German#csv", DataAccessMethod.Sequential)]
```

Comment:
I'd suggest not commenting these out.
But marking the corresponding json specs temporarily as: 
"NotSupported": "javascript,dotNet",
"NotSupportedByDesign": "python",

Then little by little specs start to pass.

Reply:
It was already done this way in other parts of the code, so, lacking any other example of how to deactivate tests that are not implemented yet in this project, I went with just deactivating them like this.
I can change it if you want, but I don't see the benefit in initializing the extractors and starting a test 27 times even if you know they are doing nothing but cluttering up your test-list on the side with skipped tests.

Reply:
The idea is that this will generate warnings in the build (skipped tests), which will make it easier to track which tests are missing.

Reply:
Ok, will change it.